packages/client/modules/teamDashboard/components/UnpaidTeamModal/UnpaidTeamModal.tsx

```diff
@@ -54,6 +69,50 @@ const UnpaidTeamModal = (props: Props) => {
   const {team} = viewer
   if (!team) return null
   const {name: teamName, organization, lockMessageHTML} = team
+
+  const {name: organizationName} = organization
+
+  const {id: orgId, billingLeaders, name: orgName} = organization
+  const [firstBillingLeader] = billingLeaders
+  const billingLeaderName = firstBillingLeader?.preferredName ?? 'Unknown'
+  const email = firstBillingLeader?.email ?? 'Unknown'
+  const isALeader = billingLeaders.findIndex((leader) => leader.id === viewerId) !== -1
+
+  const goToBilling = (upgradeCTALocation: UpgradeCTALocationEnum) => {
+    SendClientSegmentEventMutation(atmosphere, 'Upgrade CTA Clicked', {
+      upgradeCTALocation
+    })
+    history.push(`/me/organizations/${orgId}`)
+  }
+
+  if (organization.lockedAt) {
+    return (
+      <DashModal>
+        <DialogTitle>{'Organization Locked'}</DialogTitle>
+        <LockedAtContent>
+          Unfortunately, <strong>{organizationName}</strong> has exceeded the{' '}
+          {Threshold.MAX_PERSONAL_TIER_TEAMS} teams limit on the Starter Plan for more than{' '}
+          {Threshold.PERSONAL_TIER_LOCK_AFTER_DAYS} days, and your account has been locked.
+          <br />
+          <br />
+          You can re-activate your teams by upgrading your account.
+          <br />
+          <br />
+          If you’d like to keep using Parabol on the Starter Plan, please{' '}
+          <ContactUsLink href={ExternalLinks.CONTACT} target='_blank' rel='noopener noreferrer'>
+            contact us
+          </ContactUsLink>{' '}
+          to let us know which teams you’d like to delete to fit within the two-team limit.
+          {isALeader && (
+            <StyledButton size='medium' onClick={() => goToBilling('organizationLockedModal')}>
```

Comment:
+1 I don't like that non-billing users have no CTA button, not even a description to contact a billing leader

Reply:
Updated, using the same message we use for the unpaid modal

<img width="510" alt="image" src="https://user-images.githubusercontent.com/466991/212852978-23474fa4-99df-439a-b583-1d1ba93c7571.png">